### DISCUSSION TITLE
Help with merging translation

### DISCUSSION THREAD

**Discussion by kennybruhbruh** [dayanch96/BHTikTok-Plus#35] [Q&A] [answered]
Hi again, I would like to ask you if you could help me merging the translation into the BHTikTok.
I've tried unpacking deb of BHTikTokPlus and then putting lproj folder there, then repacking deb and injecting into ipa. But that resulted in broken english throughout the mod.
I know I'm missing something but don't know where to start from. Thanks

**Comment by dayanch96** [comment]
put bhtiktok.bundle into payload folder?

**Reply by kennybruhbruh** [reply]
Then it looks something like this (screenshot from BHTwitter just for reference, same thing happen in BHTikTok)

![IMG_1316](https://github.com/dayanch96/BHTikTok-Plus/assets/105291931/92b8a560-cb5c-4346-8911-ededc92ca084)

**Reply by dayanch96** [reply]
it clearly shows you that your bundle missing or you have formatting issues in your localizable.strings. what happens if you change twitter app language to english?

**Reply by kennybruhbruh** [reply]
I guess formatting issues, when I change it to English it’s fine
Could you look at the file and give me a hint? Thanks

[Localizable.strings.txt](https://github.com/dayanch96/BHTikTok-Plus/files/14089300/Localizable.strings.txt)

**Reply by dayanch96** [reply, marked as answer]
well, you have to keep this format for each line: "original" = "translation";

if you want to use “” inside your translation strings, you have to use `\"` instead of regular `"`

example:
`"original" = "use \"this\" example";` (displayed text will be like _use "this" example_)

**Reply by kennybruhbruh** [reply]
Answer I was looking for, ty
